packages/scaffolding-cli/src/domain/prompt.ts

```diff
@@ -34,7 +34,7 @@ export async function runConfig(cliArgs: CliOptions): Promise<ExitMessage> {
 }
 
 const onCancel = (prompt: any) => {
-    console.log("Never stop prompting!")
+    console.log("Selecting default answer.")
```

Comment:
this is just a placeholder we should do something more here - but it's fine for now